src/components/view/pages/payment/paymentPageView.ts

```diff
@@ -77,30 +79,42 @@ export class PaymentPageView extends View<CartDataType> {
 
     public afterRender(controller: Controller): void {
         super.afterRender(controller);
-
-        let validName: boolean = false;
-        let validAddress: boolean = false;
-        let validEmail: boolean = false;
-        let validTell: boolean = false;
-        let validCardName: boolean = false;
-        let validCardNumber: boolean = false;
-        let validCardDate: boolean = false;
-        let validCardCvv: boolean = false
-
+        
+        const valid: Validation = {
```

Comment:
Please think about this name, as for me it is a set of `formFields`, but if you want to use `valid` or `validation` in this name, you don't need to use 'valid' in keys:

`const formFields = {
validName: false,
...
}` 

or 

`const formFieldsValid = {
name: false
...
}`